src/common-elements/fields-layout.ts

```diff
@@ -155,6 +155,9 @@ export const PropertiesTable = styled.table`
   border-spacing: 0;
   width: 100%;
 
+  // Extra padding for overflow scroll
+  padding-right: 32px;
```

Comment:
this is causing a double padding for tables without scroll (with container padding)

can we add an `overflow-x` property to this element in the screenshot 
![image](https://github.com/user-attachments/assets/032ebe34-ec65-46c9-9f8e-f9589e2a6ede)

This will preserve the padding on both sides from the container while providing a scroll for the table

![image](https://github.com/user-attachments/assets/bf081525-3e27-4d17-ba0a-3def19d1ba16)

Reply:
Cool! This sounds right. I added the other overflow and removed the padding. I need to leave the other overflow because there are many combinations of places that a table can be used.